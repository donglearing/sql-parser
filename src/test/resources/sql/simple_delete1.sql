delete from T where a = 1;