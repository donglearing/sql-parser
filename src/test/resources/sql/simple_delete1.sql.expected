DELETE FROM T WHERE a = 1